enumerables.rb

```diff
@@ -0,0 +1,117 @@
+module Enumerable
+  def my_each()
```

Comment:
[optional]
If the method does not contain any argument in ruby, then you don't need to use a set of parentheses. the only my_each is enough.